students/MichaelUduhiri/Session02/fizzbuzz.py

```diff
@@ -0,0 +1,17 @@
+'''Write a program that prints the numbers from 1 to 100 inclusive.
```

Comment:
fizzbuzz looks good -- giving you credit for this.

Reply:
thanks